integration_tests/emukit/bayesian_optimization/test_local_penalization_loop.py

```diff
@@ -0,0 +1,27 @@
+import GPy
+
+from emukit.bayesian_optimization.acquisitions import ExpectedImprovement
+from emukit.bayesian_optimization.loops import BayesianOptimizationLoop
+from emukit.experimental_design import RandomDesign
+from emukit.model_wrappers import GPyModelWrapper
+from emukit.test_functions import branin_function
+
+
+def test_local_penalization():
```

Comment:
I think this is a perfect integration test - it puts a few pieces of the library together and runs end to end. Can you move it there?

Reply:
It's already in the integration tests folder

Reply:
where was i looking? sorry